### ISSUE TITLE
NullPointerException for scenarios that have no description

### ISSUE THREAD

**Issue by tfnico** [rmpestano/cukedoctor#66]
I'm currently using gulp-protractor to run the tests with protractor-cucumber-framework, and the resulting json does not contain any description on the scenario level.

Example json:

```
[
  {
    "description": "As a foo...",
    "elements": [
      {
        "id": "user does something",
        "keyword": "Scenario",
        "line": 6,
        "name": "User browses to the site successfully",
        "steps": [
          {
            "arguments": [],
            "keyword": "Given ",
            "name": "User opens a browser",
            "result": {
              "status": "passed",
              "duration": 425101
            },
            "line": 9,
            "match": {
              "location": "/Users/tfnico/projects/project/src/test-web/steps/sampleEntry_steps.js:15"
            }
          }
        ],
        "tags": [],
        "type": "scenario"
      }
    ],
    "id": "user-does-something",
    "keyword": "Feature",
    "line": 1,
    "name": "Do something",
    "tags": [],
    "uri": "/Users/tfnico/projects/project/src/test-web/gherkin/do_something.feature"
  }
]
```

Here's the stacktrace:

```
java -jar ~/Downloads/cukedoctor-main-1.0.2.jar -p build/cucumber-results -o build/cucumber-results -f all
Generating living documentation with args:
-f: all
-p: build/cucumber-results
-t: Living Documentation
-o: build/cucumber-results
Found 1 feature(s)
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Exception in thread "main" java.lang.NullPointerException
        at com.github.cukedoctor.renderer.CukedoctorScenarioRenderer.renderScenario(CukedoctorScenarioRenderer.java:93)
        at com.github.cukedoctor.renderer.CukedoctorFeatureRenderer.renderFeatureScenario(CukedoctorFeatureRenderer.java:75)
        at com.github.cukedoctor.renderer.CukedoctorFeatureRenderer.renderFeatureScenarios(CukedoctorFeatureRenderer.java:59)
        at com.github.cukedoctor.renderer.CukedoctorFeatureRenderer.renderFeature(CukedoctorFeatureRenderer.java:50)
        at com.github.cukedoctor.converter.CukedoctorConverterImpl.renderFeatures(CukedoctorConverterImpl.java:188)
        at com.github.cukedoctor.converter.CukedoctorConverterImpl.renderDocumentation(CukedoctorConverterImpl.java:98)
        at com.github.cukedoctor.CukedoctorMain.execute(CukedoctorMain.java:166)
        at com.github.cukedoctor.CukedoctorMain.execute(CukedoctorMain.java:138)
        at com.github.cukedoctor.CukedoctorMain.main(CukedoctorMain.java:149)
```

It would be nice if cukedoctor would gracefully null-check and skip the description [here](https://github.com/rmpestano/cukedoctor/blob/master/cukedoctor-converter/src/main/java/com/github/cukedoctor/renderer/CukedoctorScenarioRenderer.java#L93).

**Comment by rmpestano** [comment]
Hi @tfnico,

thanks for the heads up.

reopening the issue cause tests are failling, I'll investigate and I get some time.

**Comment by rmpestano** [comment]
should be fixed now, you can grab [1.0.3-SNAPSHOT here](https://oss.sonatype.org/content/repositories/snapshots/com/github/cukedoctor/cukedoctor-main/1.0.3-SNAPSHOT/cukedoctor-main-1.0.3-20160721.021327-8.jar).

**Comment by tfnico** [comment]
@rmpestano Thanks! It works now.
